Frontend/src/FacialRecognition/FacialRecognitionModal.jsx

```diff
@@ -2,57 +2,66 @@ import React, { useState, useRef } from 'react';
+    const [error, setError] = useState(null);
+
     const capture = () => {
-        const imageSrc = webcamRef.current.getScreenshot();
+    const imageSrc = webcamRef.current.getScreenshot();
+    if (imageSrc) {
         setImgSrc(imageSrc);
+        setError("Image Taken Correctly, you can close the screen.");
+        onImageCaptured(imageSrc);
+    } else {
+        setError("Error Taking Image, try again.");
+    }
     };
+
     const deleteImage = () => {
-        setImgSrc(null);
+    setImgSrc(null);
+    setError(null);
     };
 
-    
-
     return (
-        <Modal show={true} onHide={onClose} centered size="lg">
-            <Modal.Header closeButton>
-                <Modal.Title>Stand in front of the camera and look forward</Modal.Title>
-            </Modal.Header>
-            <Modal.Body>
-                {!imgSrc && (
-                    <Webcam
-                        audio={false}
-                        ref={webcamRef}
-                        screenshotFormat="image/jpeg"
-                        style={{ width: '100%' }}
-                    />
-                )}
-                {imgSrc && (
-                    <div style={{ position: 'relative', textAlign: 'center' }}>
-                        <img src={imgSrc} alt="Captured" style={{ width: '100%' }} />
-                        <Button
-                            variant="danger"
-                            style={{ position: 'absolute', top: '10px', right: '10px' }}
-                            onClick={deleteImage}
-                        >
-                            Delete
-                        </Button>
-                    </div>
-                )}
-            </Modal.Body>
-            <Modal.Footer>
-                {!imgSrc && (
-                    <Button variant="primary" onClick={capture}>
-                        Take Picture
-                    </Button>
-                )}
-                <Button variant="secondary" onClick={onClose}>
-                    Close
-                </Button>
-            </Modal.Footer>
-        </Modal>
+    <Modal show={true} onHide={onClose} centered size="lg">
+        <Modal.Header closeButton>
+        <Modal.Title>Stand in front of the camera and look forward</Modal.Title>
+        </Modal.Header>
+        <Modal.Body>
+        {!imgSrc && (
+            <Webcam
+            audio={false}
+            ref={webcamRef}
+            screenshotFormat="image/jpeg"
+            style={{ width: '100%' }}
+            />
+        )}
+        {imgSrc && (
+            <div style={{ position: 'relative', textAlign: 'center' }}>
+            <img src={imgSrc} alt="Captured" style={{ width: '100%' }} />
+            <Button
+                variant="danger"
+                style={{ position: 'absolute', top: '10px', right: '10px' }}
+                onClick={deleteImage}
+            >
+                Delete
+            </Button>
+            </div>
+        )}
```

Comment:
You can reorganize the code here to use the ternary operator for cleaner logic

`{
imgSrc ? ( // insert happy flow content here ) : ( // insert content for null state here)
}`